src/screens/Home/Home.tsx

```diff
@@ -1,8 +1,9 @@
 import { Box } from "@mui/material";
 import { Link } from "react-router-dom";
-import "./home.css";
+import Footer from "../../shared/components/footer/Footer";
```

Comment:
We tsconfig'd it so you can just use shared/components/footer/Footer if you pulled and rebase from main recently